resources/assets/components/InboxItem/index.js

```diff
@@ -2,7 +2,23 @@ import React from 'react';
 import { map } from 'lodash';
 import { calculateAge } from '../../helpers';
 
+import Tags from '../Tags';
+
 class InboxItem extends React.Component {
+  constructor () {
+    super();
+    this.state = {
+      showTags: false,
+    };
+    this._onAcceptClick = this._onAcceptClick.bind(this);
+  }
+
+  _onAcceptClick() {
+    this.setState({
+      showTags: true,
```

Comment:
This doesn't need to happen in this PR, but it may eventually make more sense to save `status: 'accepted'` in the state, and then conditionally render things based on that (since the thing you're mutating is the status, and whether or not tags appear in the interface is just a side-effect of that).